<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerPatientPositioningFixedBeamAxisWidget</class>
 <widget class="QWidget" name="qSlicerPatientPositioningFixedBeamAxisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_TranslationToFixedBeamAxis">
     <property name="text">
      <string>Translation to fixed beam axis</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_FromTableTopCenter">
        <property name="title">
         <string>From table top center</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="ctkCoordinatesWidget" name="CoordinatesWidget_FromTableTopCenterTranslation">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_2">
        <property name="title">
         <string>From isocenter</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="ctkCoordinatesWidget" name="CoordinatesWidget_FromIsocenterTranslation"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Frame basis</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="RadioButton_FixedReference">
             <property name="text">
              <string>FixedReference</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">ButtonGroup_FrameBasisTranslation</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RadioButton_TableTop">
             <property name="text">
              <string>TableTop</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">ButtonGroup_FrameBasisTranslation</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_2">
     <property name="text">
      <string>Fixed beam camera orientation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="PushButton_Left">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qSlicerPatientPositioningModule.qrc">
            <normaloff>:/Icons/go-previous.png</normaloff>:/Icons/go-previous.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="PushButton_Right">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qSlicerPatientPositioningModule.qrc">
            <normaloff>:/Icons/go-next.png</normaloff>:/Icons/go-next.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="PushButton_Up">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qSlicerPatientPositioningModule.qrc">
            <normaloff>:/Icons/go-up.png</normaloff>:/Icons/go-up.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="PushButton_Down">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qSlicerPatientPositioningModule.qrc">
            <normaloff>:/Icons/go-down.png</normaloff>:/Icons/go-down.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerPatientPositioningModule.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="ButtonGroup_FrameBasisTranslation"/>
 </buttongroups>
</ui>
